src/features/areas/components/PublicAreaMap.tsx

```diff
@@ -396,6 +420,15 @@ const PublicAreaMap: FC<PublicAreaMapProps> = ({ area }) => {
           place={selectedPlace}
         />
       )}
+      {showCard && (
+        <PlaceCard
```

Comment:
The name `PlaceCard` feels very ambiguous to me. Correct me if I'm wrong, but this component is only used for creating places, right? So maybe something like `CreatePlaceCard` or so would be more precise?

Reply:
Maybe `NewPlaceCard` or `DraftPlaceCard` could be other good options.